ui/lib/apps/TopSQL/pages/List/ListDetail/ListDetailContent.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useState } from 'react'
+import { Space } from 'antd'
+
+import formatSql from '@lib/utils/sqlFormatter'
+import {
+  Descriptions,
+  Expand,
+  Pre,
+  HighlightSQL,
+  TextWithInfo,
+  Card,
+  CopyLink,
+} from '@lib/components'
+import type { PlanRecord } from './ListDetailTable'
+import type { SQLRecord } from '../ListTable'
+
+interface ListDetailContentProps {
+  sqlRecord: SQLRecord
+  planRecord?: PlanRecord
+}
+
+export function ListDetailContent({
+  sqlRecord,
+  planRecord,
+}: ListDetailContentProps) {
+  const [sqlExpanded, setSqlExpanded] = useState(false)
+  const toggleSqlExpanded = () => setSqlExpanded((prev) => !prev)
+  const [planExpanded, setPlanExpanded] = useState(false)
+  const togglePlanExpanded = () => setPlanExpanded((prev) => !prev)
```

Comment:
In slow query and statement UI, these expanded state are memorized in the local storage.